ui-components/src/page_components/listing/listing_sidebar/LeasingAgent.tsx

```diff
@@ -53,6 +58,19 @@ const LeasingAgent = (props: LeasingAgentProps) => {
           officeHours={listing.leasingAgentOfficeHours}
         />
       )}
+
+      {props.managementCompany && (
+        <>
+          <div className="mt-5">
+            <p>{props.managementCompany.name}</p>
+            {props.managementCompany.website && (
```

Comment:
What's the point of checking whether the website field exists above if you guard this section by its existence here? I think you could probably remove the check / setting it above.

Reply:
Ah yeah, good point - removed the part setting it above

Reply:
Actually, I switched back to setting managementWebsite above (since it's a couple of lines and isn't as readable inline below). I realized that if a website address already has "http://", it messes everything up to prepend "http://" again, so now I check before prepending.